site/_docs/themes.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: docs
+title: Themes
+permalink: /docs/themes/
+---
+
+Jekyll has an extensive theme system, which allows you to leverage community-maintained templates and styles to customize your site's presentation. Jekyll themes package layouts, includes, stylesheets, and static assets in a way that can be overridden by your site's content.
+
+## Installing a theme
+
+1. To install a theme, first, add the theme to your site's `Gemfile`:
+
+        gem 'my-awesome-jekyll-theme'
+
+2. Save the changes to your `Gemfile`
+3. Run the command `bundle install` to install the theme
+4. Finally, activate the theme by adding the following to your site's `_config.yml`:
+
+        theme: my-awesome-jekyll-theme
+
+You can have multiple themes listed in your site's Gemfile, but only one theme can be selected in your site's `_config.yml`.
+{: .note .info }
+
+## Overriding theme defaults
+
+Jekyll themes set default layouts, includes, stylesheets and static assets, that can be overridden by your site's content. For example, if your selected theme has a `page` layout, you can override the theme's layout by creating your own `page` layout in the `_layouts` folder (e.g., `_layouts/page.html`).
+
+Jekyll will look first to your site's content, before looking to the theme's defaults, for any requested file in the following folders:
+
+* `/_layouts`
+* `/_includes`
+* `/_sass`
+* `/assets/`
+
+Refer to your selected theme's documentation and source repository for more information on what files you can override.
+{: .note .info}
+
+## Creating a theme
+
+Jekyll themes are distributed as Ruby gems. The only required file is the [Ruby Gemspec](http://guides.rubygems.org/specification-reference/). Here's an example of a minimal Gemspec for the `my-awesome-jekyll-theme` theme, saved as `/my-awsome-jekyll-theme.gemspec`:
+
+{% highlight ruby %}
+Gem::Specification.new do |s|
+  s.name     = 'My Awesome theme'
+  s.version  = '0.1.0'
+  s.license  = 'MIT'
+  s.summary  = 'This is an awesome Jekyll theme!'
+  s.author   = 'Dr. Jekyll'
+  s.email    = 'doc@jekyllrb.com'
```

Comment:
we should probably make this blank or invalid so folks don't release with these values (inevitably someone will)